demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/wwwroot/js/demo.js

```diff
@@ -267,12 +270,6 @@ function setupContextControls() {
         updateContext(event.target.value);
     });
 
-    const currentTournamentSelectEle = document.getElementById('currentTournament');
-    currentTournamentSelectEle.selectedIndex = ramdomizeSelectedOption(currentTournamentSelectEle);
-    currentTournamentSelectEle.addEventListener('change', (event) => {
-        updateContext(null, event.target.value);
-    });
-
     const deviceSelectEle = document.getElementById('device');
     deviceSelectEle.selectedIndex = ramdomizeSelectedOption(deviceSelectEle);
     deviceSelectEle.addEventListener('change', (event) => {

```

Comment:
Remove the second `null`(tournament) from updateContext in `deviceSelectEle` listener and `UseUserAgentEle` listener